Dockerfile

```diff
@@ -8,19 +8,10 @@ ENV PHPMYADMIN_DBPORT 3306
 # a special database for phpMyAdmin for advanced features.
 
 RUN apt-get update -q -q && \
- apt-get install phpmyadmin --yes --force-yes && \
+ apt-get install phpmyadmin php7.2-mbstring --yes && \
```

Comment:
Move this `php` requirement to php image.

Also please do not remove `--force-yes` for now. It is easier to see diff. It is just a warning.

Reply:
ok. should I rebase and clobber the commit where I did this, or just do a new commit reverting it?

Reply:
I do not care too much about that.